src/cometh/rental-protocol/rental-started-handler.ts

```diff
@@ -0,0 +1,88 @@
+import { log } from "@graphprotocol/graph-ts";
+import { Nft, Rental } from "../../../generated/schema";
+import { generateNftId } from "../../utils/misc";
+import { COMETHSPACESHIP } from "../../utils/constants";
+import { RentalStarted } from "../../../generated/ComethRentalProtocol/ComethRentalProtocol";
+import { SPACESHIP_ADDRESS } from "../../utils/addresses";
+
+/**
+ *     event RentalStarted(
+ *        uint256 indexed nonce,
+ *        address indexed lender,
+ *        address indexed tenant,
+ *        address token,
+ *        uint256 tokenId,
+ *        uint64 duration,
+ *        uint16 basisPoints,
+ *        uint256 start,
+ *        uint256 end
+ *    );
+ */
+export function handleRentalStarted(event: RentalStarted): void {
+  if (event.params.token.toHexString() != SPACESHIP_ADDRESS) {
+    log.debug(
+      "[handleRentalStarted] NFT {} is not a spaceship, tx: {}, skipping...",
+      [event.params.tokenId.toString(), event.transaction.hash.toHex()]
+    );
+    return;
+  }
+
+  const nftId = generateNftId(COMETHSPACESHIP, event.params.tokenId);
+  const nft = Nft.load(nftId);
+
+  if (!nft) {
+    log.debug(
+      "[handleRentalStarted] Spaceship {} does not exist, tx: {}, skipping...",
+      [event.params.tokenId.toString(), event.transaction.hash.toHex()]
+    );
+    return;
+  }
```

Comment:
No, because Cometh Rental Protocol do not check if the token exists when creating a rental offer. So we gonna have non-existent tokens in rental offers